app/assets/javascripts/application.js

```diff
@@ -16,22 +16,6 @@
 //= require jquery
 //= require jquery_ujs
 //= require bootstrap
-//= require codemirror
-//= require codemirror/modes/ruby
-//= require codemirror/modes/haskell
```

Comment:
Could we add those js files in the vendor directory as minified js's? 

Otherwise, we are adding a lot of overhead to laboratory's minifier during build, making deployments slower.

Also, code climate is going to detect hundreds of errors which we shoudn't care about.